### PR TITLE
Forgot to handle errors in test

### PR DESCRIPTION
**Pull request by honzajavorek**

### CODE REVIEW

File: test/unit/handle-runtime-problems-test.coffee [hunk 1]
```diff
@@ -62,7 +62,7 @@ describe('handleRuntimeProblems()', ->
     beforeEach((done) ->
       prepareData(apiDescriptionDocument, filename, (err, data) ->
         error = handleRuntimeProblems(data)
```
Comment (freaz) [inline test/unit/handle-runtime-problems-test.coffee]: WIll `handleRuntimeProblem` survive if `data === undefined`?

Comment (honzajavorek) [inline test/unit/handle-runtime-problems-test.coffee]: I should stop doing before thinking. Happens to me a lot lately :neutral_face: (this is genuine neutral face, not Dredd's smile)

Comment (honzajavorek) [inline test/unit/handle-runtime-problems-test.coffee]: Addressed (commit amended).

**Comment by honzajavorek** [comment]
@freaz said (in person) it's okay now.

**Pull request merged**